src/components/BookDialog/BookDialog.tsx

```diff
@@ -23,10 +23,20 @@ const BookDialog = (oldBook: BookModel) => {
   return (
     <div className={classes['c-book-dialog']}>
       <div className={classes['c-book-dialog__container']}>
-        <InputContainer onChange={handleChange} value={book.title} label="Title" />
-        <InputContainer onChange={handleChange} value={book.author} label="Author" />
-        <InputContainer onChange={handleChange} value={book.description} label="Description" />
-        <InputContainer onChange={handleChange} value={book.numOfCopies} label="Number of Copies" />
+        <InputContainer onChange={handleChange} name="title" value={book.title} label="Title" />
+        <InputContainer onChange={handleChange} name="author" value={book.author} label="Author" />
+        <InputContainer
+          onChange={handleChange}
+          name="description"
+          value={book.description}
+          label="Description"
+        />
+        <InputContainer
+          onChange={handleChange}
+          name="numOfCopies"
+          value={book.numOfCopies}
+          label="Number of Copies"
+        />
```

Comment:
Maybe we can destructure book since you are calling `book.` all over the component